### PR TITLE
Feature/second armijo changes

### PR DESCRIPTION
**Pull request by TamiaValledor**
PTAL

### CODE REVIEW

File: src/main/cpp/commands/FullHatchLevel1.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#include "commands/SetCargoDeployerIn.h"
+#include "commands/SetCargoDeployerOut.h"
+
+FullHatchLevel1::FullHatchLevel1() {
+  // Add Commands here:
+  // e.g. AddSequential(new Command1());
+  //      AddSequential(new Command2());
+  // these will run in order.
+  AddSequential(new SetCargoDeployerOut());
+  AddSequential(new SetElevator(0));
```
Comment (duane701) [inline src/main/cpp/commands/FullHatchLevel1.cpp]: FullHatchLevel should be 1 command that takes the level param.

File: src/main/cpp/commands/FullCargoShipPosition.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/FullCargoShipPosition.h"
+#include "commands/SetCargoDeployerOut.h"
+#include "commands/SetElevator.h"
+
+FullCargoShipPosition::FullCargoShipPosition() {
+  AddSequential(new SetCargoDeployerOut());
+  AddSequential(new SetElevator(29000));
```
Comment (duane701) [inline src/main/cpp/commands/FullCargoShipPosition.cpp]: FullCargoShipPosition is the same as the other Cargo Position commands.

File: src/main/cpp/commands/FullRetrieveArmPosition.cpp [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/FullRetrieveArmPosition.h"
+#include "commands/SetCargoDeployerIn.h"
+#include "commands/PivotPosition.h"
+
+
+FullRetrieveArmPosition::FullRetrieveArmPosition() {
+  AddSequential(new SetCargoDeployerIn());
```
Comment (duane701) [inline src/main/cpp/commands/FullRetrieveArmPosition.cpp]: FullRetrieveArmPosition, should this be the a command that takes a position.

File: src/main/include/commands/FullHatchLevel1.h [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <frc/commands/CommandGroup.h>
+
+class FullHatchLevel1 : public frc::CommandGroup {
```
Comment (duane701) [inline src/main/include/commands/FullHatchLevel1.h]: FullHatchLevel1 -- This family of commands are one command.

File: src/main/cpp/OI.cpp [hunk 5]
```diff
@@ -37,6 +37,16 @@
...
+#include "commands/FullCargoLevel1.h"
+#include "commands/FullCargoLevel2.h"
+#include "commands/FullCargoLevel3.h"
+#include "commands/FullHatchLevel1.h"
+#include "commands/FullHatchLevel2.h"
+#include "commands/FullHatchLevel3.h"
+#include "commands/FullScoreArmPosition.h"
+#include "commands/FullRetrieveArmPosition.h"
+#include "commands/FullTopArmPosition.h"
+#include "commands/FullCargoShipPosition.h"
```
Comment (duane701) [inline src/main/cpp/OI.cpp]: Would have been nice to commit this file with separate commits.

**Comment by TamiaValledor** [comment]
PTAL

**Comment by duane701** [comment]
Replaced by #70

**Pull request closed without merging**